private/buf/bufformat/testdata/proto2/message/v1/message_import.golden.proto

```diff
@@ -2,11 +2,9 @@ syntax = "proto2";
 
 import "custom.proto";
 
-message Thing {
-  // Trailing comment on '}'.
-
+message Thing { // Trailing comment on '}'.
```

Comment:
This is one place where I feel like the original formatting reads a little nicer, however this does better to keep the original format of the file in place.